sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeHealthcareEntityAsyncClient.java

```diff
@@ -84,7 +84,9 @@
                 activationOperation(
                     service.healthWithResponseAsync(
                         new MultiLanguageBatchInput().setDocuments(toMultiLanguageInput(documents)),
-                        options.getModelVersion(), getNonNullStringIndexType(options.getStringIndexType()),
+                        options.getModelVersion(),
+                        getNonNullStringIndexType(options.getStringIndexType()),
+                        options.isServiceLogsDisabled(),
```

Comment:
not quite sure why it's called `isServiceLogsDisabled` since we're calling the property `disableServiceLogs`

Reply:
It is a naming convention that Java follows, If is is boolean, Java go with is****Verb. I talked to Jonathon about the naming. He is fine with it.